app/routes/account.$accountId.settings.members/route.tsx

```diff
@@ -79,19 +67,20 @@ export const action: ActionFunction = async ({ request, params }) => {
       const createAccountUserResponse = await portal.createAccountUser({
         input: {
           accountID: accountId,
-          portalAppID: appId,
           email: user_email,
           roleName: user_role as RoleName,
         },
       })
       res = Boolean(createAccountUserResponse.createAccountUser)
       message = `User invite successfully sent to ${user_email}`
       if (!res) {
-        throw new Error(`Error inviting user ${user_email} on app ${appId}`)
+        throw new Error(
+          `Error inviting user ${user_email} on account ${invitedAccountName}`,
+        )
       }
 
-      // todo: send user email with app name?
-      await sendTeamInviteEmail(user_email, appId).catch((error) => {
+      // todo: send user email with account name?
```

Comment:
yep. i guess we can remove this comment then

Reply:
✅